src/CustomerBundle/Form/Type/AddressType.php

```diff
@@ -40,6 +40,11 @@ class AddressType extends AbstractType
     protected $name;
 
     /**
+     * @var BasketInterface
+     */
+    protected $basket;
+
+    /**
```

Comment:
Maybe add an upgrade note just for anyone using directly AddressType->basket, or redefining this class?

IMO it should be a patch